contracts/credit/CreditFacadeV3.sol

```diff
@@ -707,6 +700,7 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
         uint256 forbiddenTokensMask
     ) internal returns (uint256, uint256) {
         (address token, int96 quotaChange, uint96 minQuota) = abi.decode(callData, (address, int96, uint96)); // U:[FA-34]
+        if (quotaChange == 0) revert AmountCantBeZeroException(); // U:[FA-34]
```

Comment:
@StErMi probably for completeness this line should go after `quotaChanged` is rounded; something like 

```solidity
quotaChange = quotaChange != type(int96).min
     ? quotaChange / int96(uint96(PERCENTAGE_FACTOR)) * int96(uint96(PERCENTAGE_FACTOR))
     : quotaChange;
if (quotaChange == 0) revert AmountCantBeZeroException(); // U:[FA-34]
```

Reply:
Yes, I think it makes sense